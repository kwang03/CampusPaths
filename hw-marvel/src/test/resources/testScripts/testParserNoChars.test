#Tests the parser on a empty file

#Create graph
LoadGraph myGraph noChars.csv

ListNodes myGraph