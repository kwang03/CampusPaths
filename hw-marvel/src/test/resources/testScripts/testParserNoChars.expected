#Tests the parser on a empty file

#Create graph
loaded graph myGraph

myGraph contains: